moveit_core/macros/include/moveit/macros/declare_ptr.h

```diff
@@ -0,0 +1,55 @@
+/*********************************************************************
```

Comment:
Why did you create a second file for this, instead of keeping in class_forward.h? these files are already very small